src/main/java/bowling/domain/Frames.java

```diff
@@ -0,0 +1,51 @@
+package bowling.domain;
+
+public class Frames {
+
+    protected static final int BOWLING_NORMAL_FRAMES = 9;
+    private Frame head;
```

Comment:
다른 클래스에서처럼 상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가하면 좋을 것 같습니다.
```suggestion
    protected static final int BOWLING_NORMAL_FRAMES = 9;
    
    private Frame head;
```

Reply:
엇 옙옙 수정해보겠습니다~!